pkg/querier/blocks_store_replicated_set_test.go

```diff
@@ -24,6 +24,8 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
+	"github.com/grafana/mimir/pkg/storegateway"
```

Comment:
All the mimir imports should be grouped together

Reply:
Will be fixed in a separate PR